docs/contributing.rst

```diff
@@ -109,8 +109,10 @@ to see what you’ve done, and why you did it:
       sphinx.
    6. Write tests that cover all code changes.
    7. If modifications require more than one commit, break changes into
-      smaller commits. Commits involving just the docs might use ``skip ci`` in
-      their commit message to avoid running all the tests.
+      smaller commits. Commits involving just the docs might use ``[docs only]`` in
+      their commit message to avoid running all the tests. *Very* trivial commits
```

Comment:
By my reading, above we are checking for `doc` or `skip-tests`, but here it says we looking for `docs only` or`skip ci`. I'm not actually sure if that's because I don't fully understand what's happening, or if the documentation and the implementation don't match.

Reply:
Yeah, I just used a more permissive regex than documented. Skip ci is handled by Github itself and does not need to be treated explicitly